### DISCUSSION TITLE
스터디 후기 입니다.

### DISCUSSION THREAD

**Discussion by emanwoo7** [fusiondevkr/pl900#67] [1 Q&A]
7주간 노력한다고 했는데, 아직 도달점에 도착하지 않아 아쉽지만,
의미있고, 재미있는 시간이었습니다. 같이 해준 저스틴 유님과 미나님에게 감사한 마음을 전합니다.
그리고 같이 온라인 상에서 공부했던 동기(?)들도 감사했습니다. 

https://blog.naver.com/emanu/222489930992

다시 시작하는 마음으로 한 발 한 발 나아가겠습니다. 
다음 과정을 같이하려면요^^

**Comment by justinyoo** [comment]
아앗! 비공개 글이라고 합니다!

**Comment by emanwoo7** [comment]
이웃도 없지만, 이웃 공개로 되어있었네요. ^^;
